free-programming-books.md

```diff
@@ -560,6 +560,7 @@ Original Source: [Free Programming books](http://stackoverflow.com/revisions/392
 #### Programming Paradigms
 * [Introduction to Functional Programming](http://www.cl.cam.ac.uk/teaching/Lectures/funprog-jrh-1996/) (class lectures and slides)
 * [Type Theory and Functional Programming](https://www.cs.kent.ac.uk/people/staff/sjt/TTFP/)
+* [The Practice of Programming](https://github.com/wuzhouhui/tpop) (PDF) *(by Brian W. Kernighan and Ron Pike)*
```

Comment:
https://en.wikipedia.org/wiki/The_Practice_of_Programming says that the book has already been translated into 12 languages maybe one of them is Chinese?

Also, I dont know if it is ok to make a translation of a [copyrighted](http://www.informit.com/store/practice-of-programming-9780201615869) book and slap on an [MIT license](https://github.com/wuzhouhui/tpop/blob/1717cfd3da86bf1268106ace9847386c3e41459f/LICENSE) on it is an OK thing to do. 

Need to consult @vhf